packages/create-app/templates/default-app/packages/app/src/sidebar.tsx

```diff
@@ -32,6 +33,7 @@ export const AppSidebar = () => (
     <SidebarItem icon={CreateComponentIcon} to="create" text="Create..." />
     <SidebarDivider />
     <SidebarItem icon={MapIcon} to="tech-radar" text="Tech Radar" />
+    <SidebarItem icon={AppsIcon} to="release-manager" text="Release Manager" />
```

Comment:
I think we should hold off on adding this to the template for now. It also requires a few more changes than this 😅